cpp/sophus/lie/pose3.h

```diff
@@ -97,26 +127,38 @@ class Pose3 {
           rhs.frameB());
     }
 
-    Isometry lhs_a_from_rhs_b = lhs.aFromB() * rhs.aFromB();
-    Rotation rhs_b_from_lhs_b = rhs.aFromB().rotation().inverse();
-    Tangent tangent_of_rhs_b_in_lhs_a;
-
-    // here since tangent is egocentric in the rhs_b frame, we need to rotate
-    // the the tangent from the lhs_b frame to the rhs_b frame
-
-    tangent_of_rhs_b_in_lhs_a.template head<3>() =
-        rhs_b_from_lhs_b * lhs.tangentOfBInA().template head<3>() +
-        rhs.tangentOfBInA().template head<3>();
-
-    tangent_of_rhs_b_in_lhs_a.template tail<3>() =
-        rhs_b_from_lhs_b * lhs.tangentOfBInA().template tail<3>() +
-        rhs.tangentOfBInA().template tail<3>();
+    /// for notation simplicity, let's introduce the following three frames:
+    ///
+    /// alpha  := lhs.frameA()
+    /// beta  := lhs.frameB() == rhs.frameA()
+    /// gamma  := rhs.frameB()
+
+    Isometry alpha_from_beta = lhs.aFromB();
+    Isometry beta_from_gamma = rhs.aFromB();
+    // Relative velocity of the rigid body ``beta`` (with respect to frame
+    // gamma) in frame ``beta``.
+    Tangent velocity_of_beta_wrt_gamma_in_beta = lhs.tangentInB();
```

Comment:
Isn't this velocity_of_beta_wrt_alpha_in_beta?

Reply:
good catch!